src/main/msp/msp.c

```diff
@@ -1651,7 +1651,7 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
 #endif
         // Added in MSP API 1.42
 #if defined(USE_GYRO_DATA_ANALYSE)
-        sbufWriteU8(dst, gyroConfig()->dyn_notch_range);
+        sbufWriteU8(dst, gyroConfig()->dyn_notch_max_hz);
```

Comment:
`gyroConfig()->dyn_notch_max_hz` is a uint16 - not a uint8 any more. This can't be done this way. The MSP clients won't understand this and it will be a corrupted MSP message. Typically we leave the old value in place (probably just send a 0) and add the new value on to the end. But in this case need to think about what will happen for example if somebody uses a Configurator version that's not aware of this change and how it will behave. It won't know to read the extra new uint16 and that's probably ok. But what would it do with the hardcoded value for the old range?